File: MyFirstApplicationG29/app/src/main/java/com/example/myfirstapplicationg_29/MainActivity.java
package com.example.myfirstapplicationg_29;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.EditText;
import android.widget.ImageView;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

//    public void onClickMeButton(View view){
//        Log.i("INFO", "Button Clicked");
//
//        TextView textView = (TextView) findViewById(R.id.textView);
//        EditText editTextPersonName = (EditText) findViewById(R.id.editTextPersonName);
//
//        Log.i("USER_INPUT", editTextPersonName.getText().toString());
//
//        textView.setText(editTextPersonName.getText().toString());
//
//        Toast.makeText(this, "Hi, " + editTextPersonName.getText().toString(), Toast.LENGTH_LONG).show();
//    }

    public void onSwitchCatFirst(View view){
        ImageView image = (ImageView) findViewById(R.id.imageView);
        image.setImageResource(R.drawable.cat_1);
    }

    public void onSwitchCatSecond(View view){
        ImageView image = (ImageView) findViewById(R.id.imageView);
        image.setImageResource(R.drawable.cat_2);
    }



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: Java_And_Kotlin_G29/app/src/main/java/com/example/java_and_kotlin_g29/MainActivity.kt
package com.example.java_and_kotlin_g29

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import java.util.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val myUser = User("John", "Doe", 23)
        val textView = findViewById<TextView>(R.id.textView)
        // val message = "${myUser.firstName} ${myUser.lastName}, ${myUser.age}"
        textView.text = myUser.getInfo()

        val rand = Random()
        val value: Int = rand.nextInt(50)

        val randomText = findViewById<TextView>(R.id.randomText);

        randomText.text = value.toString()
    }
}